products/zoningtaxlots/recipe.yml

```diff
@@ -12,3 +12,4 @@ inputs:
     - name: dof_dtm
     - name: dcp_zoningdistricts
     - name: dcp_zoningmapindex
+    - name: dcp_zoningtaxlots
```

Comment:
let's add an `import_as` clause like in Pluto - something to denote that this table is the previous version.

Let's also add back in the `version_env_var` - there's a bash script that sets this, which is why you were having an error before. Happy to talk about this on a call tomorrow - for now you could just run `export VERSION_PREV=2024-05-01` from your terminal and things should go smoothly

Reply:
Actually let's use the real script.

After running the `plan` command (assuming you're in products/zoningtaxlots), run
```
source ../../bash/export_recipe_env.sh recipe.lock.yml 
```

This will both set the `VERSION` and `VERSION_PREV` env vars

Reply:
@fvankrieken I got the error that `Dataset dcp_zoningtaxlots requires version env var: VERSION_PREV` when running `plan`

Reply:
@damonmcc

Reply:
(have been chatting in Teams) you've used a temporary workaround and I'm working on a fix related to getting previous versions

Reply:
With #973 merged, hopefully this can be unblocked when you rebase main into this branch!